extension/src/webviewMessageFunctions.ts

```diff
@@ -57,7 +56,7 @@ export const VOID_PROVIDERS: Record<string, WebviewMessageFunction> = {
   getSidebarLoadingStatus: handleGetSidebarLoadingStatus,
 };
 
-export const executeRemainingHandlers = (message: any, webview: vscode.Webview) => {
+export const executeRemainingHandlers = async (message: any, webview: vscode.Webview) => {
```

Comment:
should we update the `message` type to its proper type instead of `any`

Reply:
Yes, will fix!